action.yml

```
@@ -59,6 +59,10 @@ inputs:
     description: 'Whitespace separated list of files to ignore when calculating the PR size (sum of changes)'
     required: false
     default: ''
+  ignore_line_deletions:
+    description: 'Ignore lines that are deleted, thereby only considering additions.'
```

Non-blocking suggestion: Maintain consistency with the description provided in the `README.md` in order to avoid having to maintain two different definitions. In this case, this would be:

```suggestion
    description: 'Whether to ignore lines which are deleted when calculating the PR size. If set to 'true', deleted lines will be ignored.'
```

---

Yeah good call